Documentation/contributing/development/season_of_docs.rst

```diff
@@ -0,0 +1,114 @@
+observability, and security issues at scale including running some of the largest Kubernetes clusters in the world. 
+Cilium has a large community of contributors from over 50 countries around the world. Multiple companies 
+have staff paid to work on Cilium too. Cilium is an incubating project in the Cloud Native Computing Foundation 
+and is the networking dataplane for Google Kubernetes Engine and Anthos.
+
+About the Problem
+~~~~~~~~~~~~~~~~~
+
+Cloud native networking and security are complex topics that even long time practitioners struggle to master.
+Cilium documentation has grown organically over time and the project has mainly been used by advanced cloud native users. 
+The documentation lacks the structure that new, and even existing, users can rely on to quickly find the information they need. 
+In addition, users sometimes struggle to discover all of the features and functionality of Cilium because the documentation 
+doesn’t make them easily discoverable. Finally, new users often have questions about how to troubleshoot setting up their network.
+
+Improving the structure of the documentation will help new users adopt Cilium and reduce the support load on the community.
+
+Project Scope
+~~~~~~~~~~~~~
+
+The project will:
+
+#. Audit the existing document structure and inventory existing documentation
+#. Using the inventory as a guide, determine existing documentation and gaps for the top three use cases (Networking, Observability and Security)
+#. Restructure the existing documentation to surface Cilium's capabilities more prominently and determine documentation gaps of existing functionality or missing links to existing talks or presentations
+#. Simplify the “Getting Started” section by moving advanced and detailed config guides out to more appropriate sections
+#. Work with the release team to restructure the documentation and create a process for keeping the documentation up to date going forward
+
+Work that is out of scope:
+
+#. Writing tutorials or reference documentation
+
+The writer will be working closely with Liz Rice, Bill Mulligan, and other Cilium maintainers. 
+We estimate the project will take five months to complete.
+
+Measuring Success
+~~~~~~~~~~~~~~~~~
+
+Cilium currently receives issues covered in the documentation in both the Slack and through Github issues. It also receives few new 
+contributors to the documentation. We believe that this restructuring will result in fewer questions and issues already covered in 
+the documentation and more contributors to the documentation since they will know where to add it.
+
+We will track three metrics (number of questions covered in documentation in Slack, number of questions covered in documentation 
+in Github issues, and number of pull requests from new contributors) monthly after the documentation is published.
+
+We would consider the project successful if, after publication of the new documentation each of the following happens:
```

Comment:
I presume that these success criteria are something like monthly rates at the beginning & end of the project? Do they account for existing trends?

Reply:
Yes, we'll need to make some estimate of this before the project starts (but we can do that after we've made the application to GSoD)